Seravo/seravo-plugin
# Sanitize SVG and allow all users upload SVG images

#### What are the main changes in this PR?
SVG images are sanitized and all users are allowed to upload SVG images.

##### Why are we doing this? Any context or related work?
Related issue: #356 

#### Manual testing steps?
1. Get or create an SVG image and add some malicious content to it.
2. Log in to your WP admin dashboard and upload the image to media library.
3. SSH to your site, find the downloaded SVG from uploads, check the file contents and see that the malicious content has been removed.
(4. If the sanitization wasn't successful, see an error message displayed after trying to upload image.)
